webpack.utils.js

```diff
@@ -37,6 +37,7 @@ function modernize( config, additionalScripts = {}, additionalPlugins = [] ) {
 				'@': path.resolve( __dirname, 'src/' ),
 			},
 		},
+		watchOptions: Object.keys( watchOptions ).length ? watchOptions : config.watchOptions,
```

Comment:
merge instead of override, consistent with how the other params work
```suggestion
		watchOptions: { ...config.watchOptions, ...watchOptions },
```